scripts/fetch/fetch.states.R

```diff
@@ -4,7 +4,7 @@
 #' 
 library(maps)
 library(sp)
-proj.string <- "+proj=laea +lat_0=45 +lon_0=-100 +x_0=0 +y_0=0 +a=6370997 +b=6370997 +units=m +no_defs"
+proj.string <- '+init=epsg:3086'
```

Comment:
Is there a reason we don't just use web mercator? I appreciate the appropriate proj, but it might simplify things to be able to pull in web mercator tiles from google and the like.